xml/bp_troubleshooting_osds.xml

```diff
@@ -158,13 +164,15 @@
     <itemizedlist>
       <listitem>
         <para>
+<!--    Which configuration file are we talking about?     -->
```

Comment:
@jschmid1 I'd presume the ceph.conf at this point, which I'm not sure we'd want to encourage this as a course of action anymore?

Reply:
I *assume* we talked about salt files here. In cephadm times we don't explicitly list devices anymore so maybe whole section can disappear and is being replaced by the drivegroup/osdspec troubleshooting guide (which doesn't really exist, yet :))

I'd get rid of this for now..

Reply:
Removed :)